lib/src/widgets/custom_image_crop_widget.dart

```diff
@@ -37,22 +37,27 @@ class CustomImageCrop extends StatefulWidget {
   /// we've provided two default painters as inspiration
   /// `DottedCropPathPainter.drawPath` and
   /// `SolidCropPathPainter.drawPath`
-  const CustomImageCrop({
+  CustomImageCrop({
     required this.image,
     required this.cropController,
     this.overlayColor = const Color.fromRGBO(0, 0, 0, 0.5),
     this.backgroundColor = Colors.white,
     this.shape = CustomCropShape.Circle,
     this.cropPercentage = 0.8,
     this.drawPath = DottedCropPathPainter.drawPath,
+    Paint? imagePaintDuringCrop,
     Key? key,
-  }) : super(key: key);
+  }) : super(key: key) {
```

Comment:
Could you move this to the constructor and thus remove the late final

Reply:
You mean like this? `this.imagePaintDuringCrop = const (Paint()..filterQuality = FilterQuality.high),`
> The constructor being called isn't a const constructor.
> Try removing 'const' from the constructor invocation

Or like this? `this.imagePaintDuringCrop = (Paint()..filterQuality = FilterQuality.high),`
> The default value of an optional parameter must be constant

Reply:
Yes